packages/app-bridge/src/AppBridgePlatformApp.ts

```diff
@@ -0,0 +1,43 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { ErrorMessageBus, IMessageBus, MessageBus } from './utilities/MessageBus';
+import { generateRandomString, notify, subscribe } from './utilities';
+import { MethodsListKeys, PlatformAppMethods, Topic } from './types';
+import { InitializationError } from './errors/InitializationError';
+import { getQueryParameters } from './utilities/queryParams';
+
+const PUBSUB_CHECKSUM = generateRandomString();
```

Comment:
This code is a side-effect and shouldn't be there.

Reply:
I think we used https://github.com/Frontify/brand-sdk/blob/9860a50edfeb15b1f5a412b778f61321b3430173/packages/app-bridge/src/AppBridgeCreateAsset.ts#L7 as an inspiration, as the `.subscribe()` method expects a token. I would be happy to improve it if there's a better way; what do you propose?